From 64d522d1bbe4778e55bea0e21bbb83f51ad39b5a Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 12 Nov 2018 21:47:53 -0700
Subject: [PATCH 06/12] msm8974-common: sepolicy: Resolve rild denials

* avc: denied { read } for name="databases" dev="mmcblk0p26" ino=33260
  scontext=u:r:rild:s0 tcontext=u:object_r:system_data_file:s0
  tclass=lnk_file permissive=1
* This is /data/data/com.android.providers.telephony/databases, which
  we need to symlink to the proper location due to Samsung libsec-ril

Change-Id: I28d9aee89dbe442bac745a6c43368a7f71e2edf6
---
 sepolicy/common/rild.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sepolicy/common/rild.te b/sepolicy/common/rild.te
index 6bbe2cf..0df7c83 100644
--- a/sepolicy/common/rild.te
+++ b/sepolicy/common/rild.te
@@ -3,6 +3,7 @@ set_prop(rild, net_radio_prop)
 allow rild radio_data_file:dir rw_dir_perms;
 allow rild radio_data_file:file create_file_perms;
 allow rild radio_data_file:lnk_file read;
+allow rild system_data_file:lnk_file read;
 
 allow rild proc_net:file w_file_perms;
 allow rild sysfs_sec:file rw_file_perms;
-- 
2.17.1

